hivemind-host/CMakeLists.txt

```diff
@@ -2,9 +2,12 @@ set(LIB_NAME "swarmus-pheromones-hivemind-host")
 set(LIB_ALIAS "SwarmUS::Pheromones::HiveMind::Host")
 
 nanopb_generate_cpp(PROTO_SRCS PROTO_HDRS
-  generic.proto
-  function-call.proto
-  message.proto
+    RELPATH
```

Comment:
Question: what is the use of the RELPATH keyword here?

Reply:
Allows nanopb to compile relative to this path and not the current one where the call is done. So your imports are relative to where the protofiles are insteand of "proto/file.proto"